client/src/websocket/actions.ts

```diff
@@ -99,6 +99,15 @@ export const openWebSocket = () => (dispatch, getState) => {
               dispatch(setColor(x, y, r, g, b));
               dispatch({ type: CanvasActionTypes.UpdatePixelError });
             }
+          } else if (status == 429) {
+            // If the user's cooldown hasn't expired yet
+            let remainingTime = json.remainingTime;
```

Comment:
nit: what's the point of this local variable? can't we just ref `json.remainingTime`?